### DISCUSSION TITLE
Google Drive search?

### DISCUSSION THREAD

**Discussion by jamie9090** [Flow-Launcher/Flow.Launcher#2399] [Q&A]
Has anyone figured out how to search Google Drive (file stream) via flow launcher? Alfred seems to be able to do it on mac, so  thought I would float it :)

**Comment by cyberjj999** [comment]
Hi. Any update for this?

**Comment by jjw24** [comment]
As far as I know, no plugin available for this yet. Potentially use Google Drive API to implement this, could also check if already client wrapper available on GitHub.
https://developers.google.com/drive/api/guides/about-sdk#what_can_you_do_with_the

**Comment by helasadf** [comment]
If you're looking to search Google Drive more efficiently, https://cloudmounter.net/mount-google-drive.html might be a useful solution. It allows you to mount Google Drive as a local drive, so you can access your files through your system’s file explorer and search directly within it. This offers a seamless way to search through all your Google Drive files, much like any other local storage on your Mac or Windows device.

**Comment by deefrawley** [comment]
I use the Google Drive app (Windows 10) and it automatically creates a local drive which Flow happily searches for me using Everything. Not sure a plugin is needed here.

**Comment by sebastianlungu** [comment]
Hi it is funny to see this is still a big topic for me, has anyone implemeneted a successful workaround ? also cloudmounter weirdly does not solve this problem
